build/Targets/GenerateAssemblyInfo.targets

```diff
@@ -1,38 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Choose>
```

Comment:
Moving this block to `Versions.props` is what made to define an `AssemblyVersionAttribute` using `$(GitHeadSha)` possible in the two project files.